### ISSUE TITLE
Deploy onedata in LAN

### ISSUE THREAD

**Issue by yanminglai** [onedata/onedata#55]
Hello there!

I am wondering if it is possible to deploy onedata(zone and provider) in LAN without domain management but only with local IPs? 

@lopiola 

Thanks!

**Comment by lopiola** [comment]
Hello,

by default, we depend on "green" TLS certificates issued for specific domains. It is not recommended to use IPs due to security reasons, but it is possible to turn off cert verification completely in the environment and the IPs should work. Another downside is that you can only have single node deployments, as the IP is used as domain. To achieve that, you need to set the Erlang env variable of `ctool` library: `force_insecure_connections` to `true`, in all applications (`op_worker` and `onepanel` in Oneprovider deployment, `oz_worker` and `onepanel` in Onezone deployment). More information can be found here: https://onedata.org/#/home/documentation/stable/doc/administering_onedata/onezone_tutorial[advanced-configuration].html

Basically you need to create a file `overlay.config` with the following content:
```
[
    {ctool, [
        {force_insecure_connections, true}
    ]}
].
```
and place it under `/etc/oz_worker/overlay.config` and `/etc/oz_panel/overlay.config` within the Onezone docker, and `/etc/op_worker/overlay.config` and `/etc/op_panel/overlay.config` within the Oneprovider docker.

Again, this is definitely not recommended and risky, but if you have a closed LAN network that you fully control then it might make sense.

**Comment by yanminglai** [comment]
Hi @lopiola, 

Thank you for the detailed guides, we have succefully depoly all services in LAN. However, all functions we tested so far work well except the replication between providers seems to be malfunction

![image](https://user-images.githubusercontent.com/16668554/143410750-c0a58610-c9f1-4a74-b7b9-70ba157e540a.png)


![image](https://user-images.githubusercontent.com/16668554/143410615-dbb3be64-9c72-4225-9851-99b8a358ff1d.png)

Any thought or suggestions will be appreciated,
Cheers

**Comment by lopiola** [comment]
Hello again,

The first thing to check is connectivity on port 6665 between the services. But I'm guessing this is not the problem.

The rtransfer module (responsible for the inter-provider transfers) does not recognize the `force_insecure_connections` option and still applies TLS with cert verification. Currently it is not possible to turn off the peer verification, we may add such option in the future. You best option now is adding the following section to `op_worker`'s overlay.config:

```
[
    {rtransfer_link, [
        {ssl, [
            {use_ssl, false}
        ]}
    ]}
].
```

note that the transfers will be performed using plain HTTP connections, but it should make it work in your case.

**Comment by lopiola** [comment]
PS if you were to merge this overlay with the previous example, it would look like this:
```
[
    {ctool, [
        {force_insecure_connections, true}
    ]},
    {rtransfer_link, [
        {ssl, [
            {use_ssl, false}
        ]}
    ]}
].
```

**Comment by yanminglai** [comment]
@lopiola Sorry for the late reply, thank you for your solutions. No further question on this topic. 

cheers
